ibm/resource_ibm_app_config_feature.go

```diff
@@ -0,0 +1,519 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/appconfiguration-go-admin-sdk/appconfigurationv1"
+	"github.com/IBM/go-sdk-core/v5/core"
+)
+
+func resourceIbmIbmAppConfigFeature() *schema.Resource {
+	return &schema.Resource{
+		Create:   resourceIbmIbmAppConfigFeatureCreate,
+		Read:     resourceIbmIbmAppConfigFeatureRead,
+		Update:   resourceIbmIbmAppConfigFeatureUpdate,
+		Delete:   resourceIbmIbmAppConfigFeatureDelete,
+		Importer: &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			"guid": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "GUID of the App Configuration service. Get it from the service instance credentials section of the dashboard.",
+			},
+			"environment_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Environment Id.",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Feature name.",
+			},
+			"feature_id": {
+				Type:        schema.TypeString,
```

Comment:
feature_id shd be only computed

Reply:
this is input and correct according to the APIs.